app.py

```diff
@@ -16,6 +16,8 @@ def create_app():
 
 app = create_app()
 
+from models import CelestialBodies, Landmark
+
```

Comment:
Good catch on this, it is tricky to figure out the errors cause the commands don't give good error messages. glad to get this fixed sooner rather than later.